test/e2e/common/networking.go

```diff
@@ -31,56 +31,52 @@ var _ = ginkgo.Describe("[sig-network] Networking", func() {
 		*/
-		framework.ConformanceIt("should function for intra-pod communication: http [LinuxOnly] [NodeConformance]", func() {
-			config := framework.NewCoreNetworkingTestConfig(f, true)
+		framework.ConformanceIt("should function for intra-pod communication: http [NodeConformance]", func() {
+			config := framework.NewCoreNetworkingTestConfig(f, false)
 			for _, endpointPod := range config.EndpointPods {
 				config.DialFromTestContainer("http", endpointPod.Status.PodIP, framework.EndpointHTTPPort, config.MaxTries, 0, sets.NewString(endpointPod.Name))
 			}
 		})
 
 		/*
-			Release : v1.9
+			Release : v1.9, v1.16
 			Testname: Networking, intra pod udp
 			Description: Create a hostexec pod that is capable of curl to netcat commands. Create a test Pod that will act as a webserver front end exposing ports 8080 for tcp and 8081 for udp. The netserver service proxies are created on specified number of nodes.
 			The kubectl exec on the webserver container MUST reach a udp port on the each of service proxy endpoints in the cluster and the request MUST be successful. Container will execute curl command to reach the service port within specified max retry limit and MUST result in reporting unique hostnames.
-			This test is marked LinuxOnly since HostNetwork is not supported on other platforms like Windows.
 		*/
-		framework.ConformanceIt("should function for intra-pod communication: udp [LinuxOnly] [NodeConformance]", func() {
-			config := framework.NewCoreNetworkingTestConfig(f, true)
+		framework.ConformanceIt("should function for intra-pod communication: udp [NodeConformance]", func() {
+			config := framework.NewCoreNetworkingTestConfig(f, false)
 			for _, endpointPod := range config.EndpointPods {
 				config.DialFromTestContainer("udp", endpointPod.Status.PodIP, framework.EndpointUDPPort, config.MaxTries, 0, sets.NewString(endpointPod.Name))
 			}
 		})
 
 		/*
-			Release : v1.9
+			Release : v1.9, v1.16
 			Testname: Networking, intra pod http, from node
 			Description: Create a hostexec pod that is capable of curl to netcat commands. Create a test Pod that will act as a webserver front end exposing ports 8080 for tcp and 8081 for udp. The netserver service proxies are created on specified number of nodes.
 			The kubectl exec on the webserver container MUST reach a http port on the each of service proxy endpoints in the cluster using a http post(protocol=tcp)  and the request MUST be successful. Container will execute curl command to reach the service port within specified max retry limit and MUST result in reporting unique hostnames.
-			This test is marked LinuxOnly since HostNetwork is not supported on other platforms like Windows.
 		*/
-		framework.ConformanceIt("should function for node-pod communication: http [LinuxOnly] [NodeConformance]", func() {
-			config := framework.NewCoreNetworkingTestConfig(f, true)
+		framework.ConformanceIt("should function for node-pod communication: http [NodeConformance]", func() {
```

Comment:
DialFromNode expects to be called from a pod with host networking.
https://github.com/kubernetes/kubernetes/blob/eb145bef43fb9d9849b30e10c33e4b3bbf252e85/test/e2e/framework/networking_utils.go#L294

Reply:
I've left ``hostnetworking=true`` for tests which are using ``DialFromNode``. The other tests should be fine.